scripts/fetch-ui-assets.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
nitpick: If we know where bash lives (i.e. if this is only for CI) I'd prefer to enforce it rather than using `/usr/bin/env`

Reply:
It's also used for local development, to build the server w/ a bundled UI